python_modules/dagster/dagster_tests/declarative_automation_tests/automation_condition_tests/builtins/test_check_condition.py

```diff
@@ -119,7 +126,8 @@ async def test_any_checks_match_basic() -> None:
     assert result.true_subset.size == 0
 
 
-def test_all_deps_blocking_checks_passed_condition() -> None:
+@pytest.mark.parametrize("real_check", [True, False])
```

Comment:
nit they're both real checks :) Optionally could make this emitted_by_asset: True/False